ui.apps/src/main/content/jcr_root/apps/hclecomm/clientlibs/clientlib-base/.content.xml

```diff
@@ -3,4 +3,4 @@
     jcr:primaryType="cq:ClientLibraryFolder"
     allowProxy="{Boolean}true"
     categories="[hclecomm.base]"
-    embed="[core.wcm.components.accordion.v1,core.wcm.components.tabs.v1,core.wcm.components.carousel.v1,core.wcm.components.image.v2,core.wcm.components.breadcrumb.v2,core.wcm.components.search.v1,core.wcm.components.form.text.v2,hclecomm.grid,hclecomm.title,hclecomm.text,hclecomm.button,hclecomm.accordion,hclecomm.image,hclecomm.carousel]"/>
+	embed="[core.wcm.components.accordion.v1,core.wcm.components.tabs.v1,core.wcm.components.carousel.v1,core.wcm.components.image.v2,core.wcm.components.breadcrumb.v2,core.wcm.components.search.v1,core.wcm.components.form.text.v2,hclecomm.grid,hclecomm.title,hclecomm.text,hclecomm.button,hclecomm.accordion,hclecomm.image,hclecomm.teaser,hclecomm.carousel]"/>
```

Comment:
@Rajamouli-hcl : is there a spacae added here ? pl remove

Reply:
spaces removed please check